docs/tutorials/excel-tutorial.md

```diff
@@ -0,0 +1,956 @@
+
+2. Open the file index.html.
+
+3. Below the `div` that contains the `sort-table` button, add the following markup:
+
+    ```html
+    <div class="padding">
+        <button class="ms-Button" id="create-chart">Create Chart</button>
+    </div>
+    ```
+
+4. Open the app.js file.
+
+5. Below the line that assigns a click handler to the `sort-chart` button, add the following code:
+
+    ```js
+    $('#create-chart').click(createChart);
+    ```
+
+6. Below the `sortTable` function add the following function.
+
+    ```js
+    function createChart() {
+        Excel.run(function (context) {
+
+            // TODO1: Queue commands to get the range of data to be charted.
+
+            // TODO2: Queue command to create the chart and define its type.
+
+            // TODO3: Queue commands to position and format the chart.
+
+            return context.sync();
+        })
+        .catch(function (error) {
+            console.log("Error: " + error);
+            if (error instanceof OfficeExtension.Error) {
+                console.log("Debug info: " + JSON.stringify(error.debugInfo));
+            }
+        });
+    }
+    ```
+
+7. Replace `TODO1` with the following code. Note that in order to exclude the header row, the code uses the `Table.getDataBodyRange` method to get the range of data you want to chart instead of the `getRange` method.
+
+    ```js
+    const currentWorksheet = context.workbook.worksheets.getActiveWorksheet();
+    const expensesTable = currentWorksheet.tables.getItem('ExpensesTable');
+    const dataRange = expensesTable.getDataBodyRange();
+    ```
+
+8. Replace `TODO2` with the following code. Note the following parameters:
+
+   - The first parameter to the `add` method specifies the type of chart. There are several dozen types.
+
+   - The second parameter specifies the range of data to include in the chart.
+
+   - The third parameter determines whether a series of data points from the table should be charted row-wise or column-wise. The option `auto` tells Excel to decide the best method.
+
+    ```js
+    let chart = currentWorksheet.charts.add('ColumnClustered', dataRange, 'auto');
+    ```
+
+9. Replace `TODO3` with the following code. Most of this code is self-explanatory. Note:
+   
+   - The parameters to the `setPosition` method specify the upper left and lower right cells of the worksheet area that should contain the chart. Excel can adjust things like line width to make the chart look good in the space it has been given.
+   
+   - A "series" is a set of data points from a column of the table. Since there is only one non-string column in the table, Excel infers that the column is the only column of data points to chart. It interprets the other columns as chart labels. So there will be just one series in the chart and it will have index 0. This is the one to label with "Value in €".
+
+    ```js
+    chart.setPosition("A15", "F30");
+    chart.title.text = "Expenses";
+    chart.legend.position = "right"
```

Comment:
Change to the `ChartLegendPosition` enum value.

Reply:
Agreed; we'll address this when the content of this tutorial is re-evaluated and refactored in Q1.